processing/p5.js-web-editor
# Mobile Preferences Screen Prototype

Part of the Mobile UI project. This PR implements a screen to access and control editor settings.

This PR requires #1467 to be merged

How to test:
1. Run the app locally with the `MOBILE_ENABLED` flag set to `true` (`MOBILE_ENABLED=true npm start`)
  - You should be able to access `/mobile`
  - You should be able to edit code
  - Clicking the gear icon should take you to the settings screen
  - You should be able to change settings and go back to the editor
  - Preview Screen should be working as well
2. Run the app without the flag enabled
  - It should work as usual
  - `/mobile` routes should not be accessible



I have verified that this pull request:

* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
